Source/options.cpp

```diff
@@ -719,9 +719,25 @@ void OptionEntryResolution::CheckResolutionsAreInitialized() const
 	}
 #endif
 
+	if (sizes.size() == 1) {
+		// Attempt to provide sensible options for 4:3 and the native aspect ratio
+		const int width = sizes[0].width;
+		const int height = sizes[0].height;
+		const int commonHeights[] = { 480, 540, 720, 960, 1080, 1440, 2160 };
+		for (int commonHeight : commonHeights) {
+			if (commonHeight > height)
+				break;
+			sizes.emplace_back(Size { commonHeight * 4 / 3, commonHeight });
+			if (commonHeight * width % height == 0)
+				sizes.emplace_back(Size { commonHeight * width / height, commonHeight });

```

Comment:
Should we maybe just provide them as 4/3 and let fit-to-screen deal with this case instead of having them in there explicitly? It will appear as if there is no difference in picking half of them to some users.

Reply:
I had considered that. I thought it might feel a bit odd to have only 4:3 resolutions to choose from on a widescreen device. Some resolutions won't divide evenly at the native resolution so listing the ones that do could potentially be helpful. Having them listed should give the user more freedom when experimenting with the `Fit to Screen` option, hopefully helping them discover how it works.

On the flip side, I also found it strange to see 1440x1080 in the list. Maybe it makes some sense to list it given that the game was originally 4:3, but players will almost certainly be more familiar with the 16:9 resolution at that height. It might be better to limit 4:3 resolutions up to 960p and then add some 16:9 variants starting from 720p. If we do that, we could maybe do without the native resolutions.

Ultimately I decided that it wouldn't hurt to include the native resolutions so I added them to this PR, but we can certainly take them out if you prefer. Thoughts?

Reply:
Maybe as a separate adjustment we could have it only show the unique heights with a p added when fit-to-screen is selected. The underlysning would then be the one with the most narrow width since that then hows an immediate difference when disabling fit-to-screen